ocaml/typing/env.ml

```diff
@@ -1134,6 +1158,14 @@ let find_constructor_address path env =
   | Papply _ ->
       raise Not_found
 
+let find_compilation_unit modname =
+  (* CR-someday lmaurer: This is a bit gross. Could add an
+     [mda_compilation_unit] field, but it's not clear how
+     to fill it in other cases. *)
+  match get_address (find_pers_mod modname).mda_address with
+  | Aunit cu -> cu
+  | Alocal _ | Adot _ -> assert false (* see [sign_of_cmi] *)
```

Comment:
Why is this function needed? Why not add `find_compilation_unit` to `Persistent_env` and then call that?

Reply:
Looks like it isn't needed after all, actually (thankfully). Getting rid of it.